tasks/utils/emission-disperse-bal.ts

```diff
@@ -0,0 +1,115 @@
+import https from "https"
+import { BN, simpleToExactAmount } from "@utils/math"
+
+const MTA_ADDRESS = "0xF501dd45a1198C2E1b5aEF5314A68B9006D842E0"
+const REPORT_URL = "https://raw.githubusercontent.com/balancer-labs/bal-mining-scripts/master/reports"
+
+interface DisperseDetails {
+    total: BN
+    recipients: Array<string>
+    values: Array<BN>
+}
+interface BalancerReward {
+    address: string
+    amount: string
+}
+interface BalancerDetails {
+    total: BN
+    rewards: Array<BalancerReward>
+}
+
+interface BalancerPolygonReport {
+    disperser: DisperseDetails
+    balancer: BalancerDetails
+}
+
+const log = (message: string, debugMode = false): void => {
```

Comment:
Implementing a debugger has been on my TODO list for a while. 
I usually use `debug` https://www.npmjs.com/package/debug in my other projects but I'm open to other suggestions.

Reply:
resolved with e270e136bd3e196c30033659347192d30ff84da0